### ISSUE TITLE
bulk fast5 playback with minknow 22.05.05

### ISSUE THREAD

**Issue by nick297** [LooseLab/readfish#203]
Hi, 

I'm trying to get readfish working ubuntu 20.04 with:
MinKNOW core: 5.1.0
Guppy: 6.1.5
Bream: 7.1.3
Script Configuration: 5.1.5

In the testing documentation in the readme it states "In the MinKNOW GUI, right click on a sequencing position and select Reload Scripts. Your version of MinKNOW will now playback the bulkfile rather than live sequencing."

But I can't seem to be able to do this. I have a Minknow plugged in with the configuration test cell

This is the start -> Start sequencing screen
![image](https://user-images.githubusercontent.com/4623090/173862360-d53f6653-7b19-436b-b85f-38d42afe5b23.png)
This is the sequencing overview screen
![image](https://user-images.githubusercontent.com/4623090/173862594-b78ef4fd-2445-4d99-90f3-d772f14961e6.png)
Is it possible to test playback wih this version of MinKnow?

Thanks,
Nick

**Comment by alexomics** [comment]
Hi Nick, 

This is a change in the MinKNOW UI, thank you for bringing it to my attention!

In your first screenshot there is a button, towards the top right-hand corner, with three dots arranged vertically. The `Reload Scripts` button is there.

Can I ask, what version branch of readfish are you planning on using? I know that they are all a little bit fragmented at the moment so it's very confusing and I am trying to write some up-to-date/better instructions.

Best, 
Alex

**Comment by nick297** [comment]
Hi Alex,

Thanks, I'll test that when I next have access to the machine.

I used these commands to install these branches
```bash
pip install git+https://github.com/nanoporetech/read_until_api@v3.0.0
pip install git+https://github.com/LooseLab/readfish@guppy_6_minknow_5
```
But the pinned `ont_pyguppy_client_lib==4.0.11` had dependency issues so I just used.
```bash
pip install ont_pyguppy_client_lib
```
I'm not sure if this is correct. I had readfish working really well pre-pandemic but have a fresh machine to play with now and need to reinstall with the latest MinKnow version.

Thanks,
Nick
